### PR TITLE
Fix error in sharding message flow description

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/cluster-sharding.md [hunk 1]
```diff
@@ -174,13 +174,14 @@ Where `#` is a number to distinguish between instances as there are multiple in
  1. Incoming message `M1` to `ShardRegion` instance `SR1`.
  2. `M1` is mapped to shard `S1`. `SR1` doesn't know about `S1`, so it asks the `SC` for the location of `S1`.
  3. `SC` answers that the home of `S1` is `SR1`.
- 4. `SR1` creates child actor for the entity `E1` and sends buffered messages for `S1` to `E1` child
- 5. All incoming messages for `S1` which arrive at `SR1` can be handled by `SR1` without `SC`. It creates entity children as needed, and forwards messages to them.
+ 4. `SR1` creates child actor shard `S1` and forwards the message to it.
+ 5. `S1` creates child actor for `E1` and forwards the message to it.
+ 6. All incoming messages for `S1` which arrive at `SR1` can be handled by `SR1` without `SC`.
```
Comment (raboof) [inline akka-docs/src/main/paradox/cluster-sharding.md]: :+1:

File: akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala [hunk 2]
```diff
@@ -918,9 +919,9 @@ private[akka] class ShardRegion(
                 } else shard.tell(msg, snd)
               case None => bufferMessage(shardId, msg, snd)
             }
-          case Some(ref) =>
-            log.debug("{}: Forwarding request for shard [{}] to [{}]", typeName, shardId, ref)
-            ref.tell(msg, snd)
+          case Some(shardRegionRef) =>
+            log.debug("{}: Forwarding message for shard [{}] to [{}]", typeName, shardId, shardRegionRef)
```
Comment (raboof) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala]: :+1:

**Review by raboof — APPROVED** [review]
Easier to follow overall!

**Pull request merged**